MParT/MultivariateExpansion.h

```diff
@@ -93,9 +93,9 @@ namespace mpart{
             Kokkos::fence();
         }
 
-        virtual void GradientImpl(StridedMatrix<const double, MemorySpace> const& pts,  
-                                  StridedMatrix<const double, MemorySpace> const& sens,
-                                  StridedMatrix<double, MemorySpace>              output) override
+        void GradientImpl(StridedMatrix<const double, MemorySpace> const& pts,  
```

Comment:
While not strictly necessary, I would advocate for keeping the virtual keyword across all of the `*Impl` classes.  It avoids some compiler warnings and, even though we have the `override` keyword, makes it more obvious that these functions stem from the base class.